docs/intro/glossary.md

```diff
@@ -0,0 +1,237 @@
+# Glossary
+
+This glossary provides definitions for common terminology used in the SDK,
+focusing on microgrid components, metrics, measurements, and power-related
+terms.
+
+## Common Acronyms
+
+### AC
+
+Alternating current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Alternating_current) for more details.
+
+### BMS
+
+Battery management system. See the [Wikipedia article](https://en.wikipedia.org/wiki/Battery_management_system) for more details.
+
+### CHP
+
+Combined heat and power. See the [Wikipedia article](https://en.wikipedia.org/wiki/Combined_heat_and_power) for more details.
+
+### DC
+
+Direct current. See the [Wikipedia article](https://en.wikipedia.org/wiki/Direct_current) for more details.
+
+### EV
+
+Electric vehicle. See the [Wikipedia article](https://en.wikipedia.org/wiki/Electric_vehicle) for more details.
+
+### PSC
+
+Passive sign convention. See the [Wikipedia article](https://en.wikipedia.org/wiki/Passive_sign_convention) for more details.
+
+### PV
+
+Photovoltaic. See the [Wikipedia article](https://en.wikipedia.org/wiki/Photovoltaics) for more details.
+
+In the SDK it is normally used as a synonym for [solar panel](#solar-panel).
+
+### SoC
+
+[State of charge](#state-of-charge). See the [Wikipedia article](https://en.wikipedia.org/wiki/State_of_charge) for more details.
+
+### SoP
+
+[State of power](#state-of-power).
+
+## Microgrid
```

Comment:
`Microgrid` is not an acronym. There should be a different header above this line, maybe `Commonly Used Terms`.

Reply:
Oh I see that this is at the same level as `## Common Acronyms`. This is a bit confusing IMO.

Reply:
In raw, yes, not that much when it is rendered, as there is also a navigation bar with the hierarchy.

I'm not saying is the best organization ever, it is just what we had in the wiki, I started with that. We could rethink the structure, but I consider this out of the scope of this PR.

Reply:
Fair enough.